response_operations_ui/templates/survey.html

```diff
@@ -16,7 +16,7 @@
     <div id="created_ce_message" class="panel__body">
       <h1 class="venus">Collection exercise created</h1>
       <p class="mars">
-        <a href="/surveys/{{ survey.shortName }}/{{ newly_created_period }}" id="newly_created_ce_link">View/edit {{survey.shortName}} {{newly_created_period}}</a>
+        <a href="/surveys/{{ survey.shortName.replace(' ', '') }}/{{ newly_created_period }}" id="newly_created_ce_link">View/edit {{survey.shortName}} {{newly_created_period}}</a>
```

Comment:
We should probably be using `url_for` instead of hard-coding the URLs:

```suggestion
        <a href="{{ url_for('collection_exercise_bp.view_collection_exercise', short_name=survey.shortName.replace(' ', ''), period=newly_created_period) }}" id="newly_created_ce_link">View/edit {{survey.shortName}} {{newly_created_period}}</a>
```